.github/workflows/schedule.yml

```diff
@@ -0,0 +1,151 @@
+name: SCHEDULED
+on:
+  workflow_dispatch:
+  schedule:
+    #  Exectues e2e compatibility test every Wednesday at 1:00 AM and if you need to convert to UTC+8 timezone, you need to -8 hours for cron expression
+    - cron: "0 17 * * 2"
+
+env:
+  CONTAINER_RUN_OPTIONS: " "
+  GINKGO_VERSION: "v1.16.4"
```

Comment:
Seems now the GINKGO_VERSION has been updated to 2.x, please also take a look, @wlq1212 thanks   https://github.com/kubeedge/kubeedge/blob/master/.github/workflows/main.yaml#L22C21-L22C21

cc @Shelley-BaoYue

Reply:
Yes, ginkgo version has been upgraded to 2.6.0.

Reply:
Okey, could you please update the `GINKGO_VERSION` to 2.x? @wlq1212